tests/test_items.py

```diff
@@ -0,0 +1,24 @@
+from todo_app.data.session_items import TRELLO_DOING, TRELLO_DONE, get_list, get_cards,TRELLO_TODO
+from todo_app.data.view import ViewModel
```

Comment:
It looks like this file hasn't been included so these tests fail - have you just missed adding that in git?